samples/logs.js

```diff
@@ -16,48 +16,51 @@
 
 async function writeLogEntry(logName) {
   // [START logging_write_log_entry]
-  // Imports the Google Cloud client library
   const {Logging} = require('@google-cloud/logging');
-
-  // Creates a client
   const logging = new Logging();
 
   /**
-   * TODO(developer): Uncomment the following line to run the code.
+   * TODO(developer): Uncomment the following line and replace with your values.
    */
-  // const logName = 'Name of the log to write to, e.g. my-log';
-
+      // const logName = 'my-log';
```

Comment:
If the user needs to define `logName` to run the sample, we like to keep these placeholders around.  They are only going to see what's inside the region tags!

Reply:
Ah, Github diff UI is a bit confusing. I didn't delete the placeholder. 

I just changed the logname copy from `'Name of the log to write to, e.g. my-log';` into `my-log` since the previous line also has a `TODO(developer)` comment that already gives context.